### PR TITLE
i18n(zh-cn): Create builderio.mdx

### PR DESCRIPTION
**Pull request by huyikai**
#### Description (required)

Create builderio.mdx

#### Related issues & labels (optional)

- #6677

### CODE REVIEW

File: src/content/docs/zh-cn/guides/cms/builderio.mdx [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import FileTree from '~/components/FileTree.astro';
+
+[Builder.io](https://www.builder.io/) 是一个适用于构建网站的支持拖放内容编辑的可视化 CMS。
+
+这个方案将向你展示如何不需要使用客户端 JavaScript，就能将你的 Builder 空间与 Astro 连接。
+
+
+## 先决条件
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/builderio.mdx:17]:
```suggestion
## 前期准备
```

这个可以考虑在后续PR中统一改成`前期准备`

File: src/content/docs/zh-cn/guides/cms/builderio.mdx [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+BUILDER_BLOGPOST_MODEL='blogpost'
+```
+
+现在，你应该已经可以在你的项目中使用这个 API 密钥了。
+
+:::note
+在编写本文时，[这个密钥是公开的](https://www.builder.io/c/docs/using-your-api-key#prerequisites)，因此无需担心隐藏或加密它。
+:::
+
+如果你希望你的环境变量具有 IntelliSense 功能，你可以在 `src/` 目录下创建一个 `env.d.ts` 文件，并按照以下方式配置 `ImportMetaEnv`：
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/builderio.mdx:39]:
```suggestion
如果你希望你的环境变量具有类型智能提示，你可以在 `src/` 目录下创建一个 `env.d.ts` 文件，并按照以下方式配置 `ImportMetaEnv`：
```

File: src/content/docs/zh-cn/guides/cms/builderio.mdx [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+        props: { title },
+      },
+    ]),
+  ];
+}
+const { slug } = Astro.params;
+const { title } = Astro.props;
+const builderModel = import.meta.env.BUILDER_BLOGPOST_MODEL;
+const builderAPIpublicKey = import.meta.env.BUILDER_API_PUBLIC_KEY;
+// Builder's API requires this field but for this use case the url doesn't seem to matter - the API returns the same HTML
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/builderio.mdx:288]: 这里注释也需要翻译

**Review by liruifengv — CHANGES_REQUESTED** [review]

File: src/content/docs/zh-cn/guides/cms/builderio.mdx [hunk 4]
```diff
@@ -0,0 +1,361 @@
...
+:::tip[框架集成]
+如果你的 Astro 项目中集成了 JavaScript 框架（如 Svelte，Vue 或 React），那么你可以考虑使用 [Builder 的集成功能](https://www.builder.io/m/integrations)，这是一种比直接通过 REST API 更优雅得获取数据的方法。
+:::
+
+访问你的索引路由，你将看到一个博客文章标题的链接列表！
+
+
+### 显示单个博客文章
+
+创建页面 `src/pages/posts/[slug].astro`，该页面将为每篇文章[动态生成一个页面](/zh-cn/core-concepts/routing/#动态路由)。
```
Comment (TheOtterlord) [inline src/content/docs/zh-cn/guides/cms/builderio.mdx:236]: On `main` we've updated this route to `/guides/`. You'll also need to update branch when you update this
```suggestion
创建页面 `src/pages/posts/[slug].astro`，该页面将为每篇文章[动态生成一个页面](/zh-cn/guides/routing/#动态路由)。
```

**Review by TheOtterlord — COMMENTED** [review]
Thanks for the PR! I've suggested a quick update to one of the links to keep it in check with the main branch

**Review by liruifengv — APPROVED** [review]

**Review by TheOtterlord — APPROVED** [review]

**Pull request merged**